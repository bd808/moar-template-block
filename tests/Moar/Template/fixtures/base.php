<!DOCTYPE HTML>
<?php use Moar\Template\Block as T; ?>
<html>
<head>
<title><?php T::block('title-full'); T::emptyblock('title');?> - Tests<?php T::endblock('title-full')?></title>
<?php T::emptyblock('html-head'); ?>
</head>
<body>
<div id="content"><?php T::emptyblock('content'); ?></div>
<?php T::emptyblock('body-end'); ?>
</body>
</html>
