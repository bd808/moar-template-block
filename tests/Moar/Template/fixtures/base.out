<!DOCTYPE HTML>
<html>
<head>
<title> - Tests</title>
</head>
<body>
<div id="content"></div>
</body>
</html>
